examples/CRISP/server/.env.example

```
@@ -12,10 +12,10 @@ ETHERSCAN_API_KEY=""
 # Cron-job API key to trigger new rounds
 CRON_API_KEY=1234567890 
 
-# Based on Default Anvil Deployments (Only for testing)
+# Based on Default Hardhat Deployments (Only for testing)
 ENCLAVE_ADDRESS="0xA51c1fc2f0D1a1b8494Ed1FE312d7C3a78Ed91C0"
 CIPHERNODE_REGISTRY_ADDRESS="0x610178dA211FEF7D417bC0e6FeD39F05609AD788"
-E3_PROGRAM_ADDRESS="0xc3e53F4d16Ae77Db1c982e75a937B9f60FE63690" # CRISPProgram Contract Address
+E3_PROGRAM_ADDRESS="0xc5a5C42992dECbae36851359345FE25997F5C42d" # CRISPProgram Contract Address
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E3_PROGRAM_ADDRESS key should go before the ENCLAVE_ADDRESS key

(UnorderedKey)